app/src/main/java/com/example/scoutconcordia/MapInfoClasses/ShuttleInfo.java

```diff
@@ -565,12 +566,12 @@ public double getEstimatedRouteTimeFromLoyola() {
     }
 
 
+    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
```

Comment:
What does this annotation do? Is it necessary?

Reply:
Is denotes element should only be called on the given API level or higher, in this case API 19. Our minSDK is currently set to 18 in the app.gradle file. 
We could change it, however we might have things that rely on api 18. 

![image](https://user-images.githubusercontent.com/26283932/78463693-f92b3680-76ad-11ea-91d2-1b6b3e00bad0.png)

Reply:
We could change this to minSDK 19 since we all installed API 28 when we all set up the project the first week. The minSDK has to be a lower API version than what we want to run, so even with 19 we would be fine. 

What do you think @vincentcerri @jaychenblue 

![image](https://user-images.githubusercontent.com/26283932/78464148-bff5c500-76b3-11ea-868e-16d470bdf9f5.png)

Reply:
I just noticed @JYSW380  is running API 21 based on the test results she posted below. 
So it would still work for her. If it stops working for someone, we could simply ask them to install a higher SDK version.

Reply:
Were these annotations added by you manually or generated somehow by Android Studio?

Are they necessary? If not, they should be removed.

Reply:
If we don't upgrade the minSDK to 19, then we have to keep it, because Kitkat is API 19.